user_guide_src/source/incoming/filters.rst

```diff
@@ -69,7 +69,7 @@ This is typically used to perform redirects, like in this example::
 
         if (! $auth->isLoggedIn())
         {
-            return redirect('login');
+            return redirect()->to('/login');
```

Comment:
Please encourage using `site_url()` since we have so many issues with URLs in subfolders:
```suggestion
            return redirect()->to(site_url('login'));
```

Reply:
@MGatner Thank you for your advice. Done.